pkg/kubelet/util/manager/watch_based_manager_test.go

```diff
@@ -65,6 +65,8 @@ func isSecretImmutable(object runtime.Object) bool {
 }
 
 func newSecretCache(ctx context.Context, fakeClient clientset.Interface, fakeClock clock.Clock, maxIdleTime time.Duration) *objectCache {
+	cCtx, cancel := context.WithCancel(ctx)
```

Comment:
If you don't need the original context anymore (and usually one doesn't), then you can just shadow it:

```suggestion
	ctx, cancel := context.WithCancel(ctx)
```

This prevents mistakes like using `ctx` when `cCtx` should have been used.

Reply:
Done.